sr_wear_logger/data/wear_data.yaml

```diff
@@ -0,0 +1,7 @@
+total_angles: {rh_FFJ1: 201.53553000000028, rh_FFJ2: 16.29440000000002, rh_FFJ3: 4.4365299999999985,
```

Comment:
can you add units eg. total angles (rad) and total_time (s)?

Reply:
Added